### ISSUE TITLE
Action when Contest finished

### ISSUE THREAD

**Issue by adamyordan** [adamyordan/PlatformCTFNetsos#19]
Bugs when contest finished
- [x] scoreboard freeze 
- [ ] contest history in user profile updated
- [ ] congratulation / appreciation to winners

please add other issues or to-dos

**Comment by wirabdillah** [comment]
user also can submit flag when contest finished

**Comment by rkkautsar** [comment]
@wirabdillah Actually, that's kind of a feature.Think of it as some kind of practice on Codeforces.

**Comment by wirabdillah** [comment]
@rkkautsar Indeed, but it should be in some kind of pratice session, not in the contest. May be we should make a problemset of past contest and do pratice session there.
